remotespark/RemoteSparkMagics.py

```diff
@@ -161,14 +160,12 @@ def _get_client_keys(self):
         return "Possible endpoints are: {}".format(self.client_manager.get_endpoints_list())
 
     def _send_command(self, client, command, language):
-        if language == Constants.lang_scala:
-            return client.execute_scala(command)
-        elif language == Constants.lang_pyspark:
-            return client.execute_pyspark(command)
-        elif language == Constants.lang_sql:
-            return client.execute_scala_sql(command)
-        elif language == Constants.lang_pysql:
-            return client.execute_pyspark_sql(command)
+        if language == "sql":
```

Comment:
Why does this no longer use the constants? The language that gets here is going to be scala or python if the user does not specify it in the sparkmagic option, and thus will always raise a language not supported exception.